libs/core/src/lib/shellbar/product-menu/product-menu.component.html

```diff
@@ -1,20 +1,23 @@
 <ng-template [ngIf]="items?.length > 0" [ngIfElse]="subHeader">
-
-    <button fd-button
-            fdType="transparent"
-            class="fd-shellbar__button--menu"
-            [fdMenu]="true"
-            [disabled]="disabled"
-            [fdMenuTrigger]="menu">
+    <button
+        fd-button
+        fdType="transparent"
+        class="fd-shellbar__button--menu"
+        [disabled]="disabled"
+        [fdMenuTrigger]="menu"
+    >
         <span class="fd-shellbar__title">{{ control }}</span>
+        <i class="sap-icon--megamenu fd-shellbar__button--icon"></i>
```

Comment:
Could you add role tag, or change <i> to `fd-icon` with `fd-shellbar__button--icon`?

Reply:
Added role